---
layout: page
title: Publications
page_title: Publications
permalink: /publications/
---
- Chuhanwen Sun*, <strong>Zhang Y*#</strong>. "STHD: probabilistic cell typing of single Spots in whole Transcriptome spatial data with High Definition" <strong><em>BioRxiv</em></strong> (2024) <a href="https://www.biorxiv.org/content/10.1101/2024.06.20.599803v1" target="_blank">Preprint</a>.   <a href="https://github.com/yi-zhang/STHD" target="_blank">STHD spatial transcriptomics software on GitHub</a>

- <strong>Zhang Y</strong>, Xiang G, <u>Jiang A</u>, Fan J, Wang C, Lynch A, Zeng Z, Zhang W, <u>Kang J</u>, Gu S, Wan C, Zhang B, Brown M#, Liu XS#, Meyer C#. "MetaTiME: Meta-components of the Tumor Microenvironment." <strong><em>Nature Communications</em></strong> (2023) <a href="https://www.nature.com/articles/s41467-023-38333-8" target="_blank">Paper</a>. <a href="https://twitter.com/Wings7Spread/status/1557419740552994819" target="_blank">Tweetorial on twitter</a>. <a href="https://github.com/yi-zhang/MetaTiME" target="_blank">MetaTiME software on GitHub</a>

- Yang L, Wang J, Altreuter J, Jhaveri A, Wong CJ, Song L, Fu J, Taing L, Bodapati S, Sahu A, Tokheim C, <strong>Zhang Y</strong>, Zeng Z, Bai G, Tang M, Qiu X, Long H, Michor F, Liu Y#, Liu XS#. Tutorial: integrative computational analysis of bulk RNA-sequencing data to characterize tumor immunity using RIMA. <strong><em>Nature Protocols</em></strong>. (2023), 1-11. <a href="https://doi.org/10.1038/s41596-023-00841-8">Paper</a>

- Baur B, Schreiber J, Shin J, Zhang S, <strong>Zhang Y</strong>, Manjunath M, Song JS, Noble WS, Roy S#. “Leveraging epigenomes and three-dimensional genome organization for interpreting regulatory variation." <strong><em>PLOS Computational Biology </em></strong> (2023) <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011286" target="_blank"> Paper</a> 

- Choi Y*, Erlich T*, Franque M, Rachmin I, Flesher J, Schiferle E, <strong>Zhang Y</strong>, Silva M, <u>Jiang A</u>, Dobry A, Lacher S, Freund O, Feder E, Cortez J, Ryu S,  Samuels Y, Zakka L, Azin M, Burd C, Shapless NN, Liu XS, Meyer C, Austen W, Bojovic B, Certrulo C, Mihm M, Hoon D, Demehri S, Hawryluk E, Fisher DE#. “Topical therapy for regression and melanoma prevention of congenital giant nevi.” <strong><em>Cell</em></strong> (2022). <a href="https://pubmed.ncbi.nlm.nih.gov/35561684/" target="_blank">Paper</a>

- Qiu X*, Boufaied N*, Hallal T, Feit A, Polo A de, Luoma AM, Alahmadi W, Larocque J, Zadra G, Xie Y, Gu S, Tang Q, <strong>Zhang Y</strong>, Syamala S, Seo JH, Bell C, O’Connor E, Liu Y, Schaeffer E, Karnes R, Weinmann S, Davicioni E, Morrissey C, Cejas P, Ellis L, Loda M, Wucherpfennig K, Pomerantz M, Spratt D, Corey E, Freedman M, Liu XS, Brown M, Long H#, Labbé D#. “MYC drives aggressive prostate cancer by disrupting transcriptional pause release at androgen receptor targets." </a> <strong><em>Nature Communications</em></strong>. (2022) <a href="https://www.biorxiv.org/content/10.1101/2021.04.23.441016v1" target="_blank">Paper</a>

- Zeng Z#, Wong C, Yang L, Ouardaoui N, Li D, Zhang W, Gu S, <strong>Zhang Y</strong>, Liu Y, Wang X, Fu J, Zhou L, Zhang B, Kim S, Yates K, Brown M, Freeman G, Uppaluri R, Manguso R, Liu XS#. “TISMO: syngeneic mouse tumor database to model tumor immunity and immunotherapy response. <strong><em> Nucleic Acids Research</em></strong>. (2022) <a href="https://academic.oup.com/nar/article/50/D1/D1391/6371975" target="_blank">Paper</a>

- Zhou L*, Zeng Z*, Egloff AM, Zhang F, Guo F, Campbell KM, Du P, Fu J, Zolkind P, Ma X, Zhang Z, <strong>Zhang Y</strong>, Wang X, Gu S, Riley R, Nakahori Y, Keegan J, Haddad R, Schoenfeld J, Griffith O, Manguso RT, Lederer JA, Liu XS#, Uppaluri R#. “Checkpoint blockade-induced CD8+ T cell differentiation in head and neck cancer responders.” <strong><em>Journal for ImmunoTherapy of Cancer</em></strong>. (2022) <a href="doi:10.1136/jitc-2021-004034" target="_blank">Paper</a>

- Penter L, <strong>Zhang Y</strong>, Savell A, Huang T, Cieri N, Thrash EM, Kim-Schulze S, Jhaveri A, Fu J, Ranasinghe S, Li S, Zhang W, Hathaway E, Nazzaro M, Kim H, Chen H, Thurin M, Rodig S, Severgnini M, Cibulskis C, Gabriel S, Livak K, Cutler C, Antin J, Nikiforow S, Koreth J, Ho V, SArmand P, Ritz J, Steicher H, Neuberg D, Hodi F, Gnjatic S, Soiffer R, Liu SX, David M, Bachireddy P, Wu CJ#. “Molecular and cellular features of CTLA-4 blockade for relapsed myeloid malignancies after transplantation.” <strong><em>Blood</em></strong>. (2021) <a href="https://ashpublications.org/blood/article-abstract/137/23/3212/475524/Molecular-and-cellular-features-of-CTLA-4-blockade" target="_blank">Paper</a>

- <strong>Zhang Y</strong>, Liu Y, Liu XS#. “Neural network architecture search with AMBER.” <strong><em>Nature Machine Intelligence. News & Views</em></strong>. (2021) 3:372–3. <a href="https://www.nature.com/articles/s42256-021-00350-x" target='_blank'>Paper</a>

- Gu SS*, Zhang W*, Wang X*, Jiang P*, Traugh N, Li Z, Meyer C, Stewig B, Xie Y, Bu X, Manos M, Front-Tello A, Gjini E, Lako A, Lim K, Conway J, Tewari A, Zeng Z, Sahu A, Tokleim C, Weirather J, Fu J, <strong>Zhang Y</strong>, Kroger B, Liang J, Cejas P, Freeman G, Rodig S, Long H, Gewurz B, Hodi F, Brown M#, Liu XS#. “Therapeutically Increasing MHC-I Expression Potentiates Immune Checkpoint Blockade.” <strong><em>Cancer Discovery</em></strong> (2021) 11:1524–41. <a href="https://cancerdiscovery.aacrjournals.org/content/11/6/1524" target="_blank">Paper</a>

- Manjunath M*, Yan J*, Youn Y, Drucker KL, Kollmeyer TM, McKinney AM, Zazubovich V, <strong>Zhang Y</strong>, Costello J, Eckel-Passow J, Selvin P, Jenkins R, Song JS#. “Functional analysis of low-grade glioma genetic variants predicts key target genes and transcription factors.” <strong><em>Neuro-Oncology</em></strong>. (2021) 23:638–49. <a href="https://academic.oup.com/neuro-oncology/article/23/4/638/5948532" target="_blank">Paper</a>
 
- Gu SS, Wang X, Hu X, Jiang P, Li Z, Traugh N, Bu X, Tang Q, Wang C, Zeng Z, Fu J, Meyer C, <strong>Zhang Y</strong>, Cejas P, Lim K, Wang J, Zhang W, Tokheim C, Sahu A, Xing X, Kroger B, Ouyang Z, Long H, Freeman G, Brown M, Liu XS. “Clonal tracing reveals diverse patterns of response to immune checkpoint blockade.” <strong><em>Genome Biology</em></strong>. (2020)  <a href="21:263. https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02166-1" target="_blank">Paper</a>

- Manjunath M, <strong>Zhang Y</strong>, Zhang S, Roy S, Perez-Pinera P, Song JS. “ABC-GWAS: Functional Annotation of Estrogen Receptor-Positive Breast Cancer Genetic Variants.” <strong><em>Frontiers in Genetics</em></strong>. (2020);0:730. 

- <strong>Zhang Y</strong>, Manjunath M, Yan J, Baur BA, Zhang S, Roy S, et al. “The cancer-associated genetic variant rs3903072 modulates immune cells in the tumor microenvironment.” <strong><em>Frontiers in Genetics</em></strong> (2019); <a href="doi:10.3389/fgene.2019.00754" target="_blank">Paper</a>

- <strong>Zhang Y</strong>. Functional interpretation of cancer-associated genetic variants." University of Illinois at Urbana-Champaign. (2019) <a href="https://www.ideals.illinois.edu/handle/2142/105017" target="_blank"> Paper</a>

- <strong>Zhang Y</strong>, M. Manjunath*, S. Zhang, D. Chasman, S. Roy, and J.S. Song. "Integrative genomic analysis predicts causative cis-regulatory mechanisms of the breast cancer-associated genetic variant rs4415084." <strong><em>Cancer Research</em></strong>, 78(7), 1579-1591, (2018). <a href="http://cancerres.aacrjournals.org/content/78/7/1579.long" target="_blank">Paper</a>

- <strong>Zhang Y</strong>, Manjunath M, Kim Y, Heintz J, Song JS. "SequencEnG: an Interactive Knowledge Base of Sequencing Techniques." <em><strong>Bioinformatics</strong></em>, bty794, (2019). <a href="https://doi.org/10.1093/bioinformatics/bty794" target="_blank">Paper</a>. <strong> <a href="http://education.knoweng.org/sequenceng/" target="_blank">SequencEnG page</a> </strong>]. Update 2022-10: SequencEnG has **10,985** users Worldwide!

- Manjunath M, <strong>Zhang Y</strong>, Kim Y, Yeo SH, Sobh O, Russell N, et al. "ClusterEnG: an interactive educational web resource for clustering and visualizing high-dimensional data." <strong><em>PeerJ Computer Science</em></strong> 4, e155, (2018). <a href="https://peerj.com/articles/cs-155/" target="_blank" >Paper</a>. [ <a href="http://education.knoweng.org/clustereng/" target="_blank">ClusterEnG page </a> ]

 (*co-first authors, #corresponding author. Lab members are underlined. )
